src/Toml/Printer.hs

```diff
@@ -77,7 +78,7 @@ pretty = prettyOptions defaultOptions
 
 {- | Converts 'TOML' type into 'Text' using provided 'PrintOptions' -}
 prettyOptions :: PrintOptions -> TOML -> Text
-prettyOptions options = Text.drop 1 . prettyTomlInd options 0 ""
+prettyOptions options = flip Text.snoc '\n' . Text.drop 1 . prettyTomlInd options 0 ""
```

Comment:
The addition of `\n` looks strange. Maybe you can just use `unlines` instead of `Text.intercalate "\n" ` in `prettyTomlInd`?

Reply:
I can't believe I forgot about `unlines`...

Reply:
Mmh, that might not work so easily... Everything breaks when I use `unlines` because `prettyTomlInd` is called recursively and `unlines` adds a whole bunch of unwanted newlines... `intercalate` has a much nicer behavior, but I'll play around a bit more.